src/utils/grammars/lexedMichelson.ne

```diff
@@ -0,0 +1,158 @@
+@{%
+const moo = require("moo");
```

Comment:
Any constraints and simplifying assumptions should be documented here.

Reply:
Done!